src/frontends/paddle/tests/test_models/gen_scripts/generate_pool3d.py

```diff
@@ -0,0 +1,227 @@
+# Copyright (C) 2018-2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+#
+# pool3d paddle model generator
+#
+import numpy as np
+import sys
+from save_model import saveModel
+
+data_type = "float32"
+
+
+def pool3d(name: str, x, attrs: dict):
+    import paddle
+
+    paddle.enable_static()
+
+    with paddle.static.program_guard(paddle.static.Program(), paddle.static.Program()):
+        node_x = paddle.static.data(name="x", shape=x.shape, dtype=data_type)
+        out = paddle.fluid.layers.pool3d(
```

Comment:
Since fluid api will be removed soon, shall we  use paddle.nn.functional.max_pool3d to validate this mapping ?@meiyang-intel

Reply:
Good idea, @AndPuQing can you help to use the api in paddle.nn.functional?

Reply:
Already fixed